website/docs/api/commands/cli.composite.md

```diff
@@ -46,6 +46,10 @@ Create an encoded composite definition from GraphQL [Composite Schema](https://d
 
 You can find a detailed guide on the creation of Composites [here](https://developers.ceramic.network/docs/composedb/guides/data-modeling/composites)
 
+If updating your composite, run this command with `--no-deploy`. Your GraphQL 
```

Comment:
I think we need to be clearer here about the types of updates we are talking about.  If you are updating your composite to add an entirely new model, then you do indeed want to deploy.  Also, attempting to index a model that's already indexed should be a no-op, so long as none of the custom indices have changed.  So I think we need to be more specific here that the issue is that you can't redeploy a composite to change the set of custom indices, to do that you need to manually connect to the SQL database and build the necessary indices yourself